gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -2,9 +2,8 @@
 {% set active_section = 'home' %}
 
 {% block content %}
-<div class="mdc-layout-grid">
-  <div class="mdc-layout-grid__inner home-page">
-    <img class="background-1" src="/static/img/background-1.png">
+<div class="mdc-layout-grid home-page">
+  <div class="mdc-layout-grid__inner">
     <img class="background-2" src="/static/img/background-2.png">
```

Comment:
I believe this line is also suppose to be removed (since the backgorund-2 css classes has been removed as well). Right now it's just a tiny image on the top left corner of the page.

Reply:
Absolutely. Good catch.

Done.